vendor/openvino/openvino.BUILD

```diff
@@ -0,0 +1,151 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])  # BSD/MIT-like license
+
+load("@com_intel_plaidml//bzl:template.bzl", "template_rule")
+
+cc_binary(
+    name = "benchmark_app",
+    srcs = glob([
+        "inference-engine/samples/benchmark_app/**/*.hpp",
+        "inference-engine/samples/benchmark_app/**/*.cpp",
+    ]),
+    data = [":plugins"],
+    deps = [
+        "inference_engine",
+        "mkldnn_plugin",
+    ],
+    linkstatic = False,
+)
+
+genrule(
+    name = "plugins",
+    outs = ["plugins.xml"],
+    cmd = 'echo "<ie><plugins><plugin name=\\"CPU\\" location=\\"./libmkldnn.so\\"></plugin></plugins></ie>" > $@',
+)
+
+template_rule(
+    name = "mkldnn_version",
+    substitutions = {
+        "@MKLDNN_VERSION_MAJOR@": "1",
+        "@MKLDNN_VERSION_MINOR@": "1",
+        "@MKLDNN_VERSION_PATCH@": "1",
+        "@MKLDNN_VERSION_HASH@": "afd",
+    },
+    src = "inference-engine/thirdparty/mkl-dnn/include/mkldnn_version.h.in",
+    out = "inference-engine/thirdparty/mkl-dnn/include/mkldnn_version.h",
+)
+
+cc_library(
+    name = "mkldnn_plugin",
+    alwayslink = True,
```

Comment:
nit: everywhere else we use `alwayslink = 1`

Reply:
done